modules/ROOT/pages/quick-start.adoc

```diff
@@ -212,15 +213,15 @@ class CreateDinosaurIT {
 
 }
 ----
-<1> `BonitaTests` is an annotation that register a JUnit 5 extension. It's a convenient way to set up a Bonita test class. This extension allows injecting a `BonitaTestToolkit` correctly instantiated in test methods.
+<1> `BonitaTests` is an annotation that registers a JUnit 5 extension. It's a convenient way to set up a Bonita test class. This extension allows injecting a `BonitaTestToolkit` correctly instantiated in the test methods.
 <2> The `BonitaTestToolkit`, injected by the `BonitaTestExtension`, is the main entry point to interact with the targeted Bonita platform.
-<3> The toolkit offers the possibility to retrieve an existing user on the targeted Bonta platform. These users are used to execute processes during the test scenario.
-<4> A process definition represents a deployed process on the targeted Bonita platform. It is used to start cases of a given process.
+<3> The toolkit offers the possibility to retrieve an existing user on the targeted Bonta platform. This user is used to execute processes during the test scenario.
```

Comment:
I actually prefer the generic form here (which wasn't perfect I agree). We say _Here we retrieve a user_ and then _Users are used to ..._
What do you think?  

```suggestion
<3> The toolkit offers the possibility to retrieve an existing user on the targeted Bonita platform. Such users are used to execute processes and tasks during test scenarios.
```